manifest.webapp

```diff
@@ -94,5 +94,10 @@
       "type": "io.cozy.sharings",
       "verbs": ["GET", "POST"]
     }
+  },
+  "mobile": {
+    "schema": "cozypass://",
+    "id_playstore": "io.cozy.pass",
+    "id_appstore": "cozy-pass/id1502262449"
```

Comment:
can you update the doc please? https://docs.cozy.io/en/cozy-stack/apps/#the-manifest

Reply:
Here it is: https://github.com/cozy/cozy-stack/pull/3263